src/contracts/Protocol/ClaimController.sol

```diff
@@ -36,6 +38,26 @@ contract ClaimController is Ownable, IClaimController {
         return claims[questionId];
     }
 
+    function getClaimLimit(uint256 questionId) public view returns (uint256) {
+        return claimLimits[questionId];
+    }
+
+    //------------------------------------------------------ Errors
+    error ClaimLimitReached();
+    error NeedClaimToAnswer();
+
+    // Api stuff
+    function setQuestionApi(address _questionApi) public onlyOwner {
+        questionApi = _questionApi;
+    }
+
+    // ------------------------------- Modifier
+    error NotTheApi();
```

Comment:
Maybe this modifier should be in it's own solidity file so we can import it into every controller?

Reply:
Yeah sounds good